packages/design-system/src/components/analytics/SendAnalytics.ts

```diff
@@ -47,50 +35,38 @@ interface AnalyticsEventProps {
   [additional_props: string]: unknown;
 }
 
-export function sendAnalytics(event: EventType, props: AnalyticsPayload, retry = 0): string {
-  if (window.utag && window.utag[event]) {
+const MAX_RETRIES = 3;
+const TIMEOUT = 300;
+
+window.utag = { link: console.log };
```

Comment:
[fix] I'm getting a build error that `window is not defined` at this line
Not sure if this is happening because this was moved out of a function?

Reply:
Whoops, that was for my debugging